### PR TITLE
Add missing publications and conference talks

### PR DESCRIPTION
**Pull request by itrich**

### CODE REVIEW

**Comment by itrich** [comment]
@garloff @fkr @ManuelaUr Please check if I've overseen a talk or publication you want to see listed on our website.

File: _data/conferences_de.yml [hunk 1]
```diff
@@ -31,3 +31,33 @@
...
+  slides: "BFH_20220824_Urban_Manuela_final.pdf"
+
+- date: "2022-08-24"
+  event: "52. Jahrestagung Gesellschaft für Informatik"
+  title: "Sovereign Cloud Stack: Open Source Toolbox für souveräne, föderierbare Cloud-Infrastrukturen"
+  details: "Dr. Manuela Urban"
+  slides: "20220929-GI_Jahrestagung-Urban_Manuela.pdf"
+
+- date: "2022-09-07"
+  event: "52. Jahrestagung Gesellschaft für Informatik"
```
Comment (fkr) [inline _data/conferences_de.yml:60]: Shouldn't the title be different here? eg. GXFS-Summit ?

Comment (itrich) [reply _data/conferences_de.yml]: You're absolutely right...

**Comment by ManuelaUr** [comment]
LGTM!

**Review by ManuelaUr — APPROVED** [review]

**Comment by itrich** [comment]
Let's merge this and add the Kielux talk in a separate PR. Agree @fkr ?

**Comment by fkr** [comment]
ack.

**Review by fkr — APPROVED** [review]
LGTM!

**Pull request merged**